index.html

```diff
@@ -202,72 +196,73 @@ <h2>About me</h2>
+                    <ul class="my-languages">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>HMTL</li>
+                        <li>CSS</li>
+                    </ul>
                 </div>
 
-                <div>
-                    <div class="square-card">
-                        <div class="language-img">
-                            <img src="images-2/second-icon.svg" alt="Framewokrs">
-                        </div>
-                        <h3>frameworks</h3>
-                        <ul class="my-languages">
-                            <li>React.js</li>
-                            <li>Ruby on Rails</li>
-                            <li>RSpec</li>
-                            <li>Capybara</li>
-                            <li>Selenium</li>
-                        </ul>
+                <div class="square-card">
+                    <div class="language-img">
+                        <img src="images-2/second-icon.svg" alt="Framewokrs">
                     </div>
+                    <h3>frameworks</h3>
+                    <ul class="my-languages">
+                        <li>React.js</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
                 </div>
-
-                <div>
-                    <div class="square-card last-square-card">
-                        <div class="language-img">
-                            <img src="images-2/third-icon.svg" alt="skills">
-                        </div>
-                        <h3>skills</h3>
-                        <ul class="my-languages">
-                            <li>Database Management</li>
-                            <li>Version Controle</li>
-                            <li>CLI</li>
-                            <li>WEb Development</li>
-                            <li>API Design</li>
-                        </ul>
+                <div class="square-card last-square-card">
+                    <div class="language-img">
+                        <img src="images-2/third-icon.svg" alt="skills">
                     </div>
+                    <h3>skills</h3>
+                    <ul class="my-languages">
+                        <li>Database Management</li>
+                        <li>Version Controle</li>
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                    </ul>
                 </div>
 
             </div>
 
         </div>
     </section>
-    <footer>
+    
         <div class="data">
             <div class="comment">
-                <p class="comment">I'm always interested in hearing about new projects, so if you'd like to chat please
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please
```

Comment:
- [ ] I would suggest you to work more on your contact form and the background image, kindly align your text between the bacground image not to overlap ,you can align text content to fit the original design .so this will make your website look professional 

- [ ] your design

![Screenshot 2023-06-15 185447](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/b67a0fe6-a3a0-4ee4-9fbb-5fc1ce03b00f)

- [ ] 

https://github.com/Young-Gaius/portfolio-setup/assets/92458236/cffd9b47-aa21-43d2-8503-3e7aaf208534